working-together/README.md

```diff
@@ -156,6 +156,11 @@ You can [download a double-sided printout] of these guides to display in your me
   - Let everyone know ahead of time what they can expect.
   - Create a calendar invite with links to the video call and dial-in information (if applicable)
   - Create a meeting agenda or note about what the meeting will cover.
+  - Strongly avoid vague meeting invitations. For people with anxiety, vague
```

Comment:
[Minor] How do you feel about this? I feel like it's more direct.

```suggestion
  - Avoid vague meeting invitations. For people with anxiety, vague
```

Reply:
Thanks! I decided to say "don't" instead, which I feel is direct and also carries more weight than "avoid".